OpenGL/Constructs/VBO.cs

```diff
@@ -90,27 +88,27 @@ private set
         /// <summary>
         /// The ID of the vertex buffer object.
         /// </summary>
-        public uint ID { get; private set; }
+        public uint ID { get; set; }
 
         /// <summary>
         /// The type of the buffer.
         /// </summary>
-        public BufferTarget BufferTarget { get; private set; }
+        public BufferTarget BufferTarget { get; set; }
 
         /// <summary>
         /// The size (in floats) of the type of data in the buffer.  Size * 4 to get bytes.
         /// </summary>
-        public int Size { get; private set; }
+        public int Size { get; set; }
 
         /// <summary>
         /// The type of data that is stored in the buffer (either int or float).
         /// </summary>
-        public VertexAttribPointerType PointerType { get; private set; }
```

Comment:
I think these should be marked as protected still.  We shouldn't be allowed user code to change this.